src/gov/nist/javax/sip/SipProviderImpl.java

```diff
@@ -491,8 +492,25 @@ public ServerTransaction getNewServerTransaction(Request request)
         SIPRequest sipRequest = (SIPRequest) request;
         try {
             sipRequest.checkHeaders();
-        } catch (ParseException ex) {
-            throw new TransactionUnavailableException(ex.getMessage(), ex);
+        } catch (ParseException pe) {
+            try {
+                SIPResponse badReqRes = sipRequest.createResponse(Response.BAD_REQUEST,
```

Comment:
In the bug report (https://github.com/RestComm/jain-slee/issues/111) there is a SIPP TC which was used to test this correction, but I will additionally create junit TC